src/Data/Managers/Helpers/QueryManager/ContinuousWithEnv.test.tsx

```diff
@@ -84,7 +83,7 @@ test("GIVEN QueryManager.ContinuousWithEnv WHEN environment changes THEN the api
   expect(apiHelper.pendingRequests).toHaveLength(1);
   expect(apiHelper.pendingRequests[0]).toEqual({
     method: "GET",
-    url: "/api/v2/notification?limit=100&filter.cleared=false",
+    url: "/api/v2/parameters?limit=100&1",
```

Comment:
I'm confused by the changes in this file, why are we going from notifications to parameters endpoint?